src/main/resources/db/migration/V1.1_001__message_received_precision.sql

```diff
@@ -0,0 +1,4 @@
+
+-- github issue 8 / add millisecond resolution
+alter table message modify received_date timestamp(3) not null;
```

Comment:
Yeah - Looking at flyway, it's not that those migrations are automatically selected, so there's a bit more effort on that front.  So far, the aim has been for mysql or mysql-like support..  The fact that H2 works out of the box is a win for a default impl.